cpu/msp430-common/include/assert.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2014 Martin Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @ingroup     msp430
+ * @{
+ *
+ * @file
+ * @brief       assert header for msp430.
+ * @see http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/assert.h.html
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+
+#ifndef ASSERT_H_
+#define ASSERT_H_
+
+#include <stdlib.h>
+
+#define ENABLE_DEBUG    (0)
```

Comment:
Yes. Other option would be to always have a printf in there, risking a potential hard-fault because the thread that used `assert` did not have enough stack-memory for printf.